<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>244</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>210</y>
     <width>191</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkPreset">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>86</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Use Preset</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkFilter">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Use User Filter</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>357</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="presets">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QRadioButton" name="sobel">
        <property name="text">
         <string>Sobel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="laplacian">
        <property name="text">
         <string>Laplacian</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="median">
        <property name="text">
         <string>Median</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="bilateral">
        <property name="text">
         <string>Bilateral</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="gaussian">
        <property name="text">
         <string>Gaussian</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="scharr">
        <property name="text">
         <string>Scharr</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>140</y>
     <width>111</width>
     <height>97</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="userfilter">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLineEdit" name="m11"/>
      </item>
      <item>
       <widget class="QLineEdit" name="m12"/>
      </item>
      <item>
       <widget class="QLineEdit" name="m13"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLineEdit" name="m21"/>
      </item>
      <item>
       <widget class="QLineEdit" name="m22"/>
      </item>
      <item>
       <widget class="QLineEdit" name="m23"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLineEdit" name="m31"/>
      </item>
      <item>
       <widget class="QLineEdit" name="m32"/>
      </item>
      <item>
       <widget class="QLineEdit" name="m33"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
